docs/user-guide/admin-cli.md

```diff
@@ -0,0 +1,141 @@
+admin CLI command and provide examples of their usage.
+
+
+# ls
+
+### Help
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls -h
+usage: vmdkops_admin.py ls [-h] [-c Col1,Col2,...]
+
+optional arguments:
+  -h, --help        show this help message and exit
+  -c Col1,Col2,...  Display selected columns: Choices = ['volume',
+                    'datastore', 'created-by', 'created', 'attached-to',
+                    'policy', 'capacity', 'used']
+```
+
+### List All
+
+List all properties for all docker volumes that exist on datastores accessible to the host.
+
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls
+Volume     Datastore   Created By VM  Created                   Attached To VM  Policy  Capacity  Used
+---------  ----------  -------------  ------------------------  --------------  ------  --------  -------
+large-vol  datastore1  Ubuntu_15.10   Sat Apr 16 13:34:12 2016  detached        N/A     1.00GB    25.00MB
+vol        datastore1  Ubuntu_15.10   Sat Apr 16 20:13:18 2016  detached        N/A     100.00MB  14.00MB
+```
+
+Note that the `Policy` column shows the VSAN storage policy. Since these example volumes live on a
+VMFS datastore they do not have a storage policy and show up as `N/A'.
+
+### List selected columns
+
+Show only the selected columns.
+
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py ls -c volume,datastore,attached-to
+Volume     Datastore   Attached To VM
+---------  ----------  --------------
+large-vol  datastore1  detached
+vol        datastore1  detached
+```
+
+Note that the that the choices are given in a comma separated list with no spaces, and are shown in
+the help given above with `vmdkops_admin ls -h`.
+
+# policy
+
+Create, configure and show the VSAN policy names and their corresponding VSAN policies. Also show whether or not they are in use.
+
+### Help
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py policy -h
+usage: vmdkops_admin.py policy [-h] {rm,create,ls,update} ...
+
+positional arguments:
+  {rm,create,ls,update}
+    rm                  Remove a storage policy
+    create              Create a storage policy
+    ls                  List storage policies and volumes using those policies
+    update              Update the definition of a storage policy and allVSAN
+                        objects using that policy
+
+optional arguments:
+  -h, --help            show this help message and exit
+```
+
+### Create
+
+Create a VSAN storage policy.
+
+```
+[root@localhost:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py policy create --name some-policy --content '(("proportionalCapacity" i0)("hostFailuresToTolerate" i0)'
+Successfully created policy: some-policy
+```
+
+Note that the VSAN storage policy given with `--content` is a standard VSAN storage policy. Please
+refer to the VSAN documentation for storage policy syntax.
+
```

Comment:
e.g. `--policy` flag in `esxcli vsan policy setdefault`
 https://pubs.vmware.com/vsphere-60/index.jsp?topic=%2Fcom.vmware.vcli.ref.doc%2Fesxcli_vsan.html